app/controllers/prison/feedbacks_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Prison::FeedbacksController < ApplicationController
+  before_action :authorize_prison_request
+  before_action :set_inbox_navigation_count, if: -> { current_user }
```

Comment:
I think we can avoid using a before action for this, and thereby cluttering up all the controllers with unrelated concerns.